### PR TITLE
CR-2098-fix_should_run_rule_to_remove_account

### PR DESCRIPTION
**Pull request by Imanuel-Mizrahi**
modified needed rules, to not use account as mandate

### CODE REVIEW

**Comment by Imanuel-Mizrahi** [comment]
Following this code change, all unit-tests passed with ratio = 0:https://jenkins.aws.indeni-ops.com/job/Cloudrail-unit-tests/506/

File: cloudrail/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = 'v0.0.24'
+__version__ = 'v0.0.25-beta.1'
```
Comment (alonashkenazi) [inline cloudrail/version.py:1]: you shouldn't commit beta

**Review by alonashkenazi — CHANGES_REQUESTED** [review]

File: cloudrail/version.py (hunk 1 above)
Comment (Imanuel-Mizrahi) [reply cloudrail/version.py:1]: @alonashkenazi , this was done automatically.
How can I avoid that?

Comment (alonashkenazi) [reply cloudrail/version.py:1]: commit __version__ = 'v0.0.24' back

**Review by alonashkenazi — APPROVED** [review]

**Comment by Imanuel-Mizrahi** [comment]
Tested this changes using cloudrail CLI, all affected rules listed in the ticket - passed.

**Review by Tomerva — APPROVED** [review]

File: cloudrail/knowledge/context/aws/iam/iam_password_policy.py [hunk 2]
```diff
@@ -45,7 +45,10 @@ def get_keys(self) -> List[str]:
         return [self.account]
 
     def get_name(self) -> str:
-        return self.account + ' policy'
+        if self.account:
+            return self.account + ' policy'
+        else:
+            return 'The account'
```
Comment (yi2020) [inline cloudrail/knowledge/context/aws/iam/iam_password_policy.py:51]: Should this be "the account's policy"?

**Review by yi2020 — CHANGES_REQUESTED** [review]

File: cloudrail/knowledge/context/aws/iam/iam_password_policy.py (hunk 2 above)
Comment (Imanuel-Mizrahi) [reply cloudrail/knowledge/context/aws/iam/iam_password_policy.py:51]: Worth thinking of it. we are dealing with the IAM account pass policy.
Perhaps we should change it to:
if self.account:
  return f'IAM account password policy for account {self.account}'
else:
  return 'IAM account password policy'
?

Comment (yi2020) [reply cloudrail/knowledge/context/aws/iam/iam_password_policy.py:51]: That works

Comment (Imanuel-Mizrahi) [reply cloudrail/knowledge/context/aws/iam/iam_password_policy.py]: OK @yi2020 , change made

**Review by yi2020 — APPROVED** [review]

**Pull request merged**